CONTRIBUTING.md

```diff
@@ -159,19 +169,19 @@ To update a sample:
 3) Make the necessary changes.
 4) Open a pull request against this repository with your changes.
 
-### How to Test Changes
+## How to Test Changes
 
-#### Odo
+### Odo
 
 odo v2: `odo create` and `odo push` to test devfile changes. See [odo v2 Doc](https://odo.dev/docs/2.5.0/using-odo/create-component) for more details.
 
 odo V3: `odo init` and `odo dev` to test devfile changes. See [odo v3 Doc](https://odo.dev/docs/command-reference/init) for more details.
 
-#### Che
+### Che
 
 Opening the URL `https://workspaces.openshift.com/#<repository_url>` in your browser should start a workspace where `exec` commands run successfully. Type `task+<space>` in the IDE command palette to see the list of available commands.
```

Comment:
It's also important to ensure the devfile commands are working in Eclipse Che:
> Opening the URL `https://workspaces.openshift.com/#<repository_url>` in your browser should start a workspace where `exec` commands run successfully. Type `task devfile` and press the `Enter` key in the IDE command palette to see the list of available devfile commands. Then execute each devfile commands one by one and ensure that they are executed successfully.

Reply:
@dmytro-ndp I'm pushing this suggestion

```suggestion
Opening the URL `https://workspaces.openshift.com/#<repository_url>` in your browser should start a workspace where `exec` commands run successfully. Type `task+<space>` in the IDE command palette to see the list of available commands.
```

Reply:
thank you, @thepetk !